source/sass/component/_tags.scss

```diff
@@ -0,0 +1,54 @@
+$c-tags-pill-background: var(--c-tags-pill-background, $color-background-card) !default;
+$c-tags-pill-color: var(--c-field-label-color, $color-darker) !default;
+$c-tags-pill-drop-shadow: var(--c-tags-pill-drop-shadow, drop-shadow(0 2px 2px rgba(0, 0, 0, 0.1))) !default;
```

Comment:
Use a predefined shadow size here. Avabile as sass vars, and showcased here as utilities. 

https://styleguide.test/utilities/shadows

Reply:
I merge this meanwhile so its testable live